index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Aweasome-books-ES6</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <header class="header">
+        <nav class="nav-bar">
+            <h1>Awesome Books</h1>
+            <ul class="nav-links">
+                <li class="nav-item nav-item1" id="list"><a href="#" >List</span></a></li>
+                <li class="nav-item nav-item2" id="add-new"><a href="#" >Add new</a></li>
+                <li class="nav-item nav-item3" id="contact"><a href="#" >Contact</a></li>
+            </ul>
+        </nav>
```

Comment:
- [ ] Please display the current date below the navigation bar using the Luxon library.

Reply:
Please note the project is working fine on my side, I do not know what I should change change
**My UI is like this**
<img src="
![2023-05-08 11_14_34-Aweasome-books-ES6](https://user-images.githubusercontent.com/110996112/236786017-6e4687a6-1afd-4aeb-8894-da1801adfda8.png)
"/>

- as you can see the current  date and time are displayed
- the add book function is working as intended